packages/apps-sdk/src/sdk.ts

```diff
@@ -49,11 +64,37 @@ export class SDK {
   public closeApp(): Promise<any> {
     return this.sendChannel.send(EVENT_TYPE_REMOVE_APP, { appName: `lt.${this.appName}.*` });
   }
+
   /**
    * getAppSettings: Returns a promise that resolves to the App Settings
    */
   public getAppSettings(): Promise<AppSettingsResult> {
-    return this.sendChannel.send(EVENT_TYPE_GET_APP_SETTINGS, { appName: this.appName });
+    return new Promise((resolve, reject) => {
+      return this.sendChannel.send(EVENT_TYPE_GET_APP_SETTINGS, { appName: this.appName }).then((result: any) => {
+        // Here we extend the app initialData with what we have in this.initialData
+        const extendedResult = { 
+          ...result,
+          data: { 
+            ...result.data,
```

Comment:
What is the result, why are sure that it has `data` key as it has a type `any`. What is the `data` ? Is SDK's instance ?

Reply:
You are right I should type the result